server/server.go

```diff
@@ -62,13 +62,13 @@ type server struct {
 	sessions  map[string]*session
 }
 
-func (s *server) ApplyDDL(ctx context.Context, databaseName string, stmt ast.DDL) error {
+func (s *server) ApplyDDL(ctx context.Context, databaseName string, stmt ast.DDL, parentStmt *ast.CreateTable) error {
```

Comment:
Please don't add parentStmt here. You can see if the table uses interleave option and its table name by own ast.CreateTable.

Reply:
Ah yes, but I can't know the primary keys of the parent table so I can't know which columns to use in the `FOREIGN KEY` statement right?

Reply:
You can get a table definition from database's fields by the table name.

Reply:
Oh ok, I will rewrite it.